libethash-cl/CLMiner.cpp

```diff
@@ -384,6 +380,11 @@ void CLMiner::workLoop()
 			if (nonce != 0)
 				report(nonce, current);
 
+			current = w;        // kernel now processing newest work
+            current.startNonce = startNonce;
```

Comment:
1. The indentation is off.
2. If we are going to keep the "start nonce" in `current`, maybe the free `startNonce` is not needed.